package.json

```diff
@@ -64,6 +64,7 @@
     "chai": "^4.2.0",
     "chai-exclude": "^2.0.1",
     "chai-shallow-deep-equal": "^1.4.6",
+    "chai-xml": "^0.3.2",
```

Comment:
I'm wondering how is related the new `--force` argument with this new testing tool for XML files, I see you use it later in some tests, but still not sure how those tests are related.

Reply:
Oh, I see you added them as a side fix, sorry.

Reply:
Linking the PR with https://github.com/medic/cht-core/issues/6537 though.